DCWL-2140 Amend README.md (#484)
hmrc/customs-declarations: README.md

# Customs Declarations

> This service offers a general interface for custom declarations, where those declarations are represented using the WCO declarations schema. In addition to submitting declarations, the Customs Declarations service allows you to amend, cancel and also provides the ability to upload supporting documents and make arrival notifications

Reviewer: ```suggestion
This service offers a general interface for custom declarations, where those declarations are represented using the WCO declarations schema. In addition to submitting declarations, the Customs Declarations service allows you to amend and cancel them. It also provides the ability to upload supporting documents and make arrival notifications.
```

As the notification process is asynchronous the only response to the declarant from this API is to indicate the success (or otherwise) of the validation and submission to the CDS backend.

## Development Setup
- This microservice requires mongoDB 4.+
- Run locally: `sbt run` which runs on port `9820` by default
- Run with test endpoints: `sbt 'run -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`

##  Service Manager Profiles

> The Customs Declarations application can be run locally from Service Manager using the following profiles: -

Reviewer: ```suggestion
The Customs Declarations service can be run locally from Service Manager, using the following profiles:
```

|---------------------------------------|:------------------------------------------------------------------|----------------------------------------------------------------|
| CUSTOMS_DECLARATION_ALL               | sm2 --start CUSTOMS_DECLARATION_ALL                               | To run all CDS applications.                                   |
| CUSTOMS_INVENTORY_LINKING_EXPORTS_ALL | sm2 --start CUSTOMS_INVENTORY_LINKING_EXPORTS_ALL                 | To run all CDS Inventory Linking Exports related applications. |
| CUSTOMS_INVENTORY_LINKING_IMPORTS_ALL | sm2 --start CUSTOMS_INVENTORY_LINKING_IMPORTS_ALL                 | To run all CDS Inventory Linking Imports related applications. |

## Run Tests
- Run Unit Tests: `sbt test`
- Run Integration Tests: `sbt IntegrationTest/test`
- Run Unit and Integration Tests: `sbt test IntegrationTest/test`
- Run Unit and Integration Tests with coverage report: `./run_all_tests.sh`<br/> which runs `clean scalastyle coverage test it:test coverageReport dependencyUpdates"`

### Acceptance Tests

> Run CDS ATs: see [here](https://github.com/hmrc/customs-automation-test)

Reviewer: ```suggestion
To run the CDS acceptance tests, see [here](https://github.com/hmrc/customs-automation-test)
```

### Performance Tests

> Run Performance Tests see [here](https://github.com/hmrc/customs-declaration-performance-test)

Reviewer: ```suggestion
To run performance tests, see [here](https://github.com/hmrc/customs-declaration-performance-test).
```

## API documentation

> - For Customs Declarations API documentation please see [here](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/customs-declarations)

Reviewer: ```suggestion
- For Customs Declarations API documentation, see [here](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/customs-declarations).
```
